### PR TITLE
HTTP XML Docs

### PR DESCRIPTION
**Pull request by mhoeger**
Add xml doc comments for http stuff

### CODE REVIEW

File: src/DotNetWorker/Context/Features/InvocationFeaturesExtensions.cs [hunk 1]
```diff
@@ -14,7 +14,7 @@ internal static class InvocationsFeaturesExtensions
         /// Gets a feature of type <typeparamref name="T"/> from the <see cref="IInvocationFeatures"/>.
         /// </summary>
         /// <typeparam name="T">The type of the feature to get.</typeparam>
-        /// <param name="features"><The <see cref="IInvocationFeatures"/>./param>
+        /// <param name="features"><see cref="IInvocationFeatures"/></param>
```
Comment (fabiocav) [inline src/DotNetWorker/Context/Features/InvocationFeaturesExtensions.cs:17]: I have the same fixes in my other PR 😄

File: src/DotNetWorker/Definition/BindingDirection.cs [hunk 2]
```diff
@@ -3,9 +3,18 @@
 
 namespace Microsoft.Azure.Functions.Worker
 {
```
Comment (fabiocav) [inline src/DotNetWorker/Definition/BindingDirection.cs:5]: Looks like we tackled some of the same types. The PR I mentioned earlier deals with some of these

File: src/DotNetWorker/Http/HttpCookie.cs [hunk 3]
```diff
@@ -5,30 +5,66 @@
 
 namespace Microsoft.Azure.Functions.Worker.Http
 {
+    /// <summary>
+    /// Http response cookie object
```
Comment (fabiocav) [inline src/DotNetWorker/Http/HttpCookie.cs:9]: 👍🏼

File: src/DotNetWorker/Http/HttpCookie.cs [hunk 4]
```diff
@@ -5,30 +5,66 @@
...
+        /// </summary>
+        /// <param name="name">Cookie name</param>
+        /// <param name="value">Cookie value</param>
         public HttpCookie(string name, string value)
         {
             Name = name ?? throw new ArgumentNullException(nameof(name));
             Value = value ?? throw new ArgumentNullException(nameof(value));
         }
 
+        /// <summary>
```
Comment (fabiocav) [inline src/DotNetWorker/Http/HttpCookie.cs:24]: For properties, can we use the `Gets or sets .. ` or appropriate (e.g. `Gets ...` for getter only) pattern?

**Review by fabiocav — APPROVED** [review]

**Review by brettsam — APPROVED** [review]

**Pull request merged**